addons/project/models/project.py

```diff
@@ -1079,48 +1079,48 @@ def _read_group_personal_stage_type_ids(self, stages, domain, order):
     ], default='this', store=False)
     recurrence_message = fields.Char(string='Next Recurrencies', compute='_compute_recurrence_message')
 
-    repeat_interval = fields.Integer(string='Repeat Every', default=1, compute='_compute_repeat', readonly=False)
+    repeat_interval = fields.Integer(string='Repeat Every', default=1, compute='_compute_repeat', readonly=False, compute_sudo=True)
```

Comment:
```suggestion
    repeat_interval = fields.Integer(string='Repeat Every', default=1, compute='_compute_repeat', readonly=False, groups="project.group_project_user")
```

Same suggestion for the others fields. 🙂